backend/lib/rpc-service/service.js

```diff
@@ -40,12 +42,14 @@ export const rpcRequest = async (method, params) => {
   }
 }
 /**
- * @param {object} actorEventFilter
+ * @param {{fromHeight:number,toHeight:number,fields: unknown}} actorEventFilter
  * Returns actor events filtered by the given actorEventFilter
+ * @param {MakeRpcRequest} makeRpcRequest
  * @returns {Promise<Array<Static<typeof BlockEvent>>>}
  */
 export async function getActorEvents (actorEventFilter, makeRpcRequest) {
-  const rawEvents = await makeRpcRequest('Filecoin.GetActorEventsRaw', [actorEventFilter])
+  /** @typedef {unknown[]} ActorEventsRaw */
+  const rawEvents = /** @type {ActorEventsRaw} */(await makeRpcRequest('Filecoin.GetActorEventsRaw', [actorEventFilter]))
```

Comment:
This can be simplified by inlining the type:

```suggestion
  const rawEvents = /** @type {unknown[]} */(await makeRpcRequest('Filecoin.GetActorEventsRaw', [actorEventFilter]))
```

(Nitpick - feel free to ignore.)